server/controllers/cellMove/cellMovePrisonerSearch.ts

```diff
@@ -35,17 +37,32 @@ export default ({ prisonerCellAllocationService }: Params) =>
       true,
     )
 
```

Comment:
prisoner data from prisonerCellAllocationService.getInmates() is different to the prisoner data returned by prisonerDetailsService.getPrisoners()
Both sets of data are required.